src/popup/pages/SignTransaction.vue

```diff
@@ -32,7 +32,10 @@ export default defineComponent({
         }
 
         const signedTransaction = await sdk.signTransaction(
-          transaction, { networkId, app: callbackOrigin.value },
+            transaction as `tx_${string}`, {
```

Comment:
```suggestion
            transaction as Encoded.Transaction {
```
Encoded.Transaction from SDK